server/tests/api/videos/video-transcoder.ts

```diff
@@ -285,6 +286,13 @@ describe('Test video transcoding', function () {
     this.timeout(80000)
 
     {
+      // Generate high bitrate video on the fly.
+      // https://stackoverflow.com/a/18154439
+      ffmpeg(buildAbsoluteFixturePath('video_short1.webm'))
```

Comment:
You should wrap the call to ffmpeg in a Promise

Reply:
Done, although I'm not sure what `res(imageName)` means. It gives me an error if I put that.